test/python/opflow/test_aer_pauli_expectation.py

```diff
@@ -120,6 +121,22 @@ def test_pauli_expect_state_vector(self):
 
         np.testing.assert_array_almost_equal(sampled.eval(), [0, 0, 1, -1], decimal=1)
 
+    def test_pauli_expect_non_hermitian_state_vector(self):
+        """pauli expect state vector with non hermitian operator test"""
+        states_op = ListOp([One, Zero, Plus, Minus])
+
+        op = np.zeros((2, 2))
+        op[0, 0] = 0
+        op[0, 1] = 1
+        op[1, 0] = 2
+        op[1, 1] = 3
```

Comment:
This looks like a bit much code for a matrix definition 😉 
```suggestion
        op = np.array([[0, 1], [2, 3]])
```